lib/src/bezier_tools.dart

```diff
@@ -377,3 +377,22 @@ List<Intersection> locateIntersectionsRecursively(
 
   return results;
 }
+
+/// Returns the index of the point in [points] that is closest (in terms of
+/// geometric distance) to [targetPoint].
```

Comment:
Should there be an explanation about what happens if there is a tie between two indices?  From what I gather, it will return the first index.  If so, there should be a test case for this.

For the points [-1.0, -1.0] and [1.0, 1.0] and a target point of [1.0, -1.0], the distanceSquared between each of the points and the target point should be the same.